<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdgeDetectionDialog</class>
 <widget class="QDialog" name="EdgeDetectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>641</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>19</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>结构元大小（n*n,请输入n为奇数)：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确认大小</string>
   </property>
  </widget>
  <widget class="QPushButton" name="standardButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Standard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="externalButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>630</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>External</string>
   </property>
  </widget>
  <widget class="QPushButton" name="internalButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>630</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Internal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>输入结构元（元素必须为0或1）：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
